mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -17,66 +20,42 @@ pipeline {
         stage('Build and Test') {
             steps {
                 checkout scm
-                sh '''
-                    mkdir -p build && cd build
-                    # Set the path to llvm-symbolizer
-                    PATH=$PATH:/opt/rocm-4.1.1/llvm/bin/
-
-                    cmake -G Ninja ../llvm \
-                        -DLLVM_ENABLE_PROJECTS="mlir;lld" \
-                        -DLLVM_BUILD_EXAMPLES=ON \
-                        -DLLVM_TARGETS_TO_BUILD="X86;AMDGPU" \
-                        -DCMAKE_BUILD_TYPE=Release \
-                        -DLLVM_ENABLE_ASSERTIONS=ON \
-                        -DBUILD_SHARED_LIBS=ON \
-                        -DLLVM_BUILD_LLVM_DYLIB=ON \
-                        -DMLIR_ROCM_RUNNER_ENABLED=1 \
-                        -DMLIR_MIOPEN_DRIVER_ENABLED=1 \
-                        -DMLIR_MIOPEN_DRIVER_PR_E2E_TEST_ENABLED=1 
-                        # if we open sqlite, step4 lowering failed until adding the custom db
-                        # -DMLIR_MIOPEN_SQLITE_ENABLED=1
-
-                    # Build LLVM / MLIR and run tests
-                    cmake --build . --target check-mlir
-                '''
+                cmakeBuild generator: 'Ninja',\
+                    buildDir: 'build',\
+                    buildType: 'Release',\
+                    installation: 'InSearchPath',\
```

Comment:
[Question] Is this `installations` key/value necessary? What does the `InsearchPath` do on our project?

Reply:
The `installations` key is something the CMake plugin uses to determine which CMake to use. InSearchPath means that it uses whatever CMake can be found in $PATH, and so that we control our CMake version via the Dockerfile, as opposed to by defining a particular version of Jenkins to use in the diff.

Reply:
Okay that makes a lot of sense. I was misunderstanding this as regards to the installation of our llvm project, but turns out this is just cmake build tools related.